icefoganalytics/travel-authorization
# Clean up Adminstration Users and User Edit Pages

Relates to:

- Implemented as part of investigation of https://github.com/ytgov/travel-authorization/issues/110

# Context

Users Edit page was flaky, and did not permit edit of user name. Breadcrumbs where also broken.

# Implementation

Standardize naming and location of users page and user edit page.
Fix user name update.
Fix breadrumbs.
Perform some minimal cleanup of users edit page code.
Fix user "status" badge.

# Screenshots

![image](https://github.com/user-attachments/assets/6ae96479-dfd9-4a7d-afcf-94c36a598c01)

# Testing Instructions

1. Run the test suite via `dev test` (or `dev test_api`)
2. Boot the app via `dev up`
3. Log in to the app at http://localhost:8080
4. Select "Administration" from the top right kebab menu.
5. Click on the "User Management" item in the far left.
6. Check that the User Management page now has a loading state.
7. Click on a user to go to the "User Editor" and edit the user.
8. Check that the content loads correctly, and the user status is now correct.
9. Check that edits to all the fields persist after refreshing the page.
10. Check that the breadcrumb back to the "User Management" page works.
